_faq/using_forseti/logs.md

```diff
@@ -7,7 +7,8 @@ order: 3
 The installation log is stored in `/tmp/deployment.log` on the Forseti 
```

Comment:
How do I find that?

Reply:
Clarified that this is the directory.  The following phrase `on the Forseti Compute Engine instance` will tell them that this is on the VM.  Is there something else that should be clarified?